pypowsybl/network/impl/network.py

```diff
@@ -306,6 +306,39 @@ def get_elements(self, element_type: ElementType, all_attributes: bool = False,
             result = result[attributes]
         return result
 
+    def get_sub_networks(self, all_attributes: bool = False, attributes: List[str] = None,
+                         **kwargs: ArrayLike) -> DataFrame:
+        """
+        Get a dataframe of sub networks
+
+        Args:
+            all_attributes: flag for including all attributes in the dataframe, default is false
+            attributes: attributes to include in the dataframe. The 2 parameters are mutually exclusive.
+                        If no parameter is specified, the dataframe will include the default attributes.
+
+        Returns:
+            A dataframe of sub networks.
+        """
```

Comment:
I'm not sure it is usefull but maybe a description of the column could be added 
for example 
 
Notes:
            The resulting dataframe, depending on the parameters, will include the following columns:

            - **id**: the subnetwork id

Reply:
there is no column just an index